dubbo-rpc/dubbo-rpc-webservice/pom.xml

```diff
@@ -43,10 +43,12 @@
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-frontend-simple</artifactId>
+            <version>3.2.4</version>
         </dependency>
         <dependency>
             <groupId>org.apache.cxf</groupId>
             <artifactId>cxf-rt-transports-http</artifactId>
+            <version>3.2.4</version>

```

Comment:
version is managed in here: https://github.com/apache/incubator-dubbo/blob/master/dependencies-bom/pom.xml#L184
You may like to update it in dependencies-bom project